Makefile

```diff
@@ -85,3 +85,15 @@ install-hooks:
 
 k8s_itests: .paasta/bin/activate
 	make -C k8s_itests all
+
+openapi-codegen:
```

Comment:
Can we run it from a pre-commit hook, so the generated files always will be in sync with schemas?
We also better run it from a test inside Travis to make sure that no PR with the pre-commit hook skipped will pass.

Reply:
I agree that running it from a pre-commit hook is up for discussion, but the test at Travis is a must IMO, otherwise we'll always have a drifted schema.

Reply:
travis is running all pre-commit hooks in general_itests target, i've added relevant checks to existing pre-commit hook, this should do the trick.

Reply:
I'm also wondering if we need to prevent committing changes in `paastaapi/` without having changes in schema files...